integrations/operator/embeddedtbot/config.go

```diff
@@ -0,0 +1,100 @@
+package embeddedtbot
+
+import (
+	"context"
+	"flag"
+	"strings"
+	"time"
+
+	"github.com/gravitational/trace"
+
+	"github.com/gravitational/teleport/api/types"
+	"github.com/gravitational/teleport/lib/tbot/config"
+	"github.com/gravitational/teleport/lib/tbot/identity"
+)
+
+const (
+	defaultCertificateTTL  = time.Hour
+	defaultRenewalInterval = 30 * time.Minute
```

Comment:
can we specify the defaultRenewalInterval as fraction of defaultCertificateTTL to avoid someone updating `defaultCertificateTTL` and not  `defaultRenewalInterval` ? 
like:

```go
const (
	defaultCertificateTTL  = 10*time.Minute
	defaultRenewalInterval = 30 * time.Minute
)
```

We should also enforce parameters specified by the customer

Reply:
I don't think tbot does that currently, I think it is nicer to dynamically compute renewal interval but I'd like to stay as close as possible to the tbot flags.